x11-packages/xfce4-cpugraph-plugin/0001-cpu-data-thread-stat.patch

```diff
@@ -0,0 +1,181 @@
+diff --git a/panel-plugin/os.cc b/panel-plugin/os.cc
+index a8c9e40..3016be9 100644
+--- a/panel-plugin/os.cc
++++ b/panel-plugin/os.cc
+@@ -79,9 +79,176 @@ static kstat_ctl_t *kc;
+ #endif
+ 
+ #if defined (__linux__) || defined (__FreeBSD_kernel__)
++
++#ifdef __ANDROID__
++
++#include <sys/system_properties.h>
++#include <unistd.h>
++#include <glob.h>
++
++static int _sdkver = -1;
++
++void get_sdk_ver() {
++    char sdk_ver_str[PROP_VALUE_MAX];
++    __system_property_get("ro.build.version.sdk", sdk_ver_str);
++	_sdkver = atoi(sdk_ver_str);
```

Comment:
Please fix this patch to consistently use soft or hard tabs, whatever's in the file it applies to.